tests/Integration/Pennant/FeatureDirectiveTest.php

```diff
@@ -0,0 +1,122 @@
+<?php declare(strict_types=1);
+
+namespace Tests\Integration\Pennant;
+
+use Illuminate\Testing\TestResponse;
+use Laravel\Pennant\Feature;
+use Nuwave\Lighthouse\Testing\MocksResolvers;
+use Nuwave\Lighthouse\Testing\UsesTestSchema;
+use Tests\TestCase;
+
+final class FeatureDirectiveTest extends TestCase
+{
+    use UsesTestSchema;
+    use MocksResolvers;
+
+    /** @before */
+    public function setUpSchema(): void
+    {
+        $this->schema = /* @lang GraphQL */ <<<'GRAPHQL'
+type Query {
+    fieldWhenFeatureInactive: String! 
+        @feature(name: "foo", when: INACTIVE)
+        @mock
+    fieldWhenFeatureActive: String!
+        @feature(name: "foo", when: ACTIVE)
+        @mock
+    fieldWhenFeatureActiveByDefault: String!
+        @feature(name: "foo")
+        @mock 
+}
+GRAPHQL;
```

Comment:
Please define a separate schema in each test case, experience has shown that having a single test schema per class gets unwieldy.